gio/src/unix_mount_entry.rs

```diff
@@ -9,10 +9,10 @@ use std::mem;
 
 impl UnixMountEntry {
     #[doc(alias = "g_unix_mount_at")]
-    pub fn new_at<P: AsRef<std::path::Path>>(mount_path: P) -> (UnixMountEntry, u64) {
+    pub fn new_at<P: AsRef<std::path::Path>>(mount_path: P) -> (Option<UnixMountEntry>, u64) {
         unsafe {
             let mut time_read = mem::MaybeUninit::uninit();
-            let ret = from_glib_full(ffi::g_unix_mount_at(
+            let ret = from_glib_none(ffi::g_unix_mount_at(
```

Comment:
It's `transfer full` though, or is that annotation wrong too?

Reply:
Somehow I was thinking that none=Option. I will have a look if I did this mistake in other commits :|